src/feature-flags.ts

```diff
@@ -0,0 +1,36 @@
+/** All registered GK togglers. Do not manually add items to it. */
+const GateKeeperTogglers: Record<string, () => void> = {};
+
+// Add your new feature flags below
+
+export const isRequirementDebuggerEnabled = registerFeatureFlagChecker('RequirementDebugger');
```

Comment:
I'm worried that assigning and using constants will be less scalable and searchable. Instead, would exporting a single function work?

For example, first register all the feature flag checkers. Then export something like `isFlagEnabled: flagName => boolean`, and in call it like `isFlagEnabled('RequirementDebugger')`.